### PR TITLE
integration rework testing

### PR DESCRIPTION
**Pull request by smark88**

### CODE REVIEW

**Review by snf2ye — APPROVED** [review]
LGTM and thanks for bumping the api chart version! (Reminder: Switch back to action version!)

File: .github/workflows/gradle-build-pr.yml [hunk 1]
```diff
@@ -73,46 +73,46 @@ jobs:
...
-        uses: broadinstitute/datarepo-actions@0.26.0
+        uses: broadinstitute/datarepo-actions@ms-integrationrework
         with:
           actions_subcommand: 'helmdeploy'
-          helm_secret_chart_version: '0.0.4'
-          helm_datarepo_api_chart_version: '0.0.19'
-          helm_datarepo_ui_chart_version: '0.0.9'
+          helm_create_secret_manager_secret_version: '0.0.6'
+          helm_datarepo_api_chart_version: '0.0.22'
+          helm_datarepo_ui_chart_version: '0.0.12'
```
Comment (fboulnois) [inline .github/workflows/gradle-build-pr.yml:96]: Is this just a bump to latest version, or are there other changes?

**Review by fboulnois — COMMENTED** [review]
One question:

File: .github/workflows/gradle-build-pr.yml (hunk 1 above)
Comment (smark88) [reply .github/workflows/gradle-build-pr.yml:96]: - latest charts
- uniformed helm value names
- uniformed deployments
- uniformed code for tests for api/ui 

https://github.com/broadinstitute/datarepo-helm-definitions/pull/127

**Review by nmalfroy — APPROVED** [review]

**Pull request merged**